src/lwc/rd2EntryForm/__tests__/data/recurringDonationObjectInfo.json

```diff
@@ -0,0 +1,1677 @@
+{
```

Comment:
Some of the test data in this PR could get trimmed down a bit. There's a lot of describe info that we don't need.